tsconfig.lib.esm.json

```diff
@@ -1,8 +1,8 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
-    "target": "es2020",
-    "module": "es2020",
+    "target": "es2015",
+    "module": "es2015",
```

Comment:
Didn't es2019 work?

Reply:
Its just that, from what I understood, Babel understands ES2015 by default, and full ES2017/19/20/21 support requires plugins.